frontend/src/components/ReceiveOnchain.tsx

```diff
@@ -45,7 +45,6 @@ export default function ReceiveOnchain({ onchainAddress }: { onchainAddress: str
                             <Copy copyValue={onchainAddress} />
                         </div>
                     </div>
-                    <p className="text-2xl font-light transition-all">{isCheckingAddress ? "Checking" : "Checking..."}</p>
```

Comment:
you'll have an unused variable. can make the useQuery statement above return nothing:

``` 
-- const { isLoading: isCheckingAddress } = useQuery({
++ useQuery({
```